plugins/ipqualityscore/plugin.spec.yaml

```diff
@@ -0,0 +1,666 @@
+      catch_all:
+        title: Catch All
+        description: Is this email likely to be a "catch all" where the mail server verifies all emails tested against it as valid? It is difficult to determine if the address is truly valid in these scenarios, since the email's server will not confirm the account's status.
+        type: boolean
+        required: false
+      timed_out:
+        title: Timed Out
+        description: Did the connection to the mail service provider timeout during the verification? If so, we recommend increasing the "timeout" variable above the default 7 second value. Lookups that timeout with a "valid" result as false are most likely false and should be not be trusted.
+        type: boolean
+        required: false
+      suspect:
+        title: Suspect
+        description: This value indicates if the mail server is currently replying with a temporary error and unable to verify the email address. This status will also be true for "catch all" email addresses as defined below. If this value is true, then we suspect the "valid" result may be tainted and there is not a guarantee that the email address is truly valid.
+        type: boolean
+        required: false
+      recent_abuse:
+        title: Recent Abuse
+        description: This value will indicate if there has been any recently verified abuse across our network for this email address. Abuse could be a confirmed chargeback, fake signup, compromised device, fake app install, or similar malicious behavior within the past few days.
+        type: boolean
+        required: false
+      fraud_score:
+        title: Fraud Score
+        description: The overall Fraud Score of the user based on the email's reputation and recent behavior across the IPQS threat network. Fraud Scores >= 75 are suspicious, but not necessarily fraudulent.
+        type: integer
+        required: false
+      suggested_domain:
+        title: Suggested Domain
+        description: Default value is "N/A". Indicates if this email's domain should in fact be corrected to a popular mail service. This field is useful for catching user typos. For example, an email address with "gmai.com", would display a suggested domain of "gmail.com". This feature supports all major mail service providers.
+        type: string
+        required: false
+      leaked:
+        title: Leaked
+        description: Was this email address associated with a recent database leak from a third party? Leaked accounts pose a risk as they may have become compromised during a database breach.
+        type: boolean
+        required: false
+      sanitized_email:
+        title: Sanitized Email
+        description: Sanitized email address with all aliases and masking removed, such as multiple periods for Gmail.com.
+        type: string
+        required: false
+      domain_age:
+        title: Domain Age
+        description: A human description of when this domain was registered.
+        type: object
```

Comment:
Type object is perfectly usable. If this is a defined type within the IPQualityScore service however, is it possible to define a custom type for usage here? This is completely dependant on your use case however it would allow easier access to the variables for users. This would be true for all uses of object within the plugin specification.